CONTRIBUTING.md

```diff
@@ -1,18 +1,17 @@
 # Contributing to Linuxbrew
 
-[Linuxbrew](https://github.com/Linuxbrew/brew) is a fork of [Homebrew](https://github.com/Homebrew/brew). Homebrew is merged into Linuxbrew roughly once per week. To contribute a new formula or a new version of an existing formula, please submit your pull request to Homebrew rather than to Linuxbrew. Patches to fix issues that you have reproduced on both Linuxbrew and Homebrew should be sent to Homebrew. Please send your pull request to Linuxbrew if you are in doubt.
+[Linuxbrew](https://github.com/Linuxbrew/brew) is a fork of [Homebrew](https://github.com/Homebrew/brew). Homebrew/brew is merged into Linuxbrew/brew roughly once per week. If you have access to a macOS system and are able to test your changes there: please submit your pull request to Homebrew rather than to Linuxbrew. If not, please submit your pull requests to Linuxbrew. Patches to fix issues that you have reproduced on both Linuxbrew and Homebrew on macOS should be sent to Homebrew. Please send your pull request to Linuxbrew if you are in doubt.
 
-Patches to fix issues particular to Linux should not affect the behaviour of the formula on Mac. Use `if OS.mac?` and `if OS.linux?` as necessary to preserve the existing behaviour on Mac.
+Patches to fix issues particular to Linux should not affect the behaviour of the formula on macOS. Use `if OS.mac?` and `if OS.linux?` as necessary to preserve the existing behaviour on macOS.
 
-# Contributing to Homebrew
-First time contributing to Homebrew? Read our [Code of Conduct](https://github.com/Homebrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct).
+# Contributing to Linuxbrew
+First time contributing to Linuxbrew? Read our [Code of Conduct](https://github.com/Linuxbrew/brew/blob/master/CODEOFCONDUCT.md#code-of-conduct).
 
 ### Report a bug
 
 * run `brew update` (twice)
 * run and read `brew doctor`
-* read [the Troubleshooting Checklist](http://docs.brew.sh/Troubleshooting.html)
```

Comment:
Can we point https://github.com/Linuxbrew/brew/blob/master/docs/Troubleshooting.md instead of simply deleting this line?